_database-integrations/mssql/google-cloud/v1/google-cloud-v1.md

```diff
@@ -0,0 +1,174 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/databases/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Google Cloud SQL Server
+keywords: google-cloudsql-microsoft-sql-server, database integration, etl google-cloudsql-microsoft-sql-server, google-cloudsql-microsoft-sql-server etl
+permalink: /integrations/databases/google-cloudsql-microsoft-sql-server
+summary: "Connect and replicate data from your Google Cloud SQL Server database using Stitch's Google Cloud SQL Server integration."
```

Comment:
I think this should be changed to reflect the fact that users must use the `Microsoft SQL Server` integration to actually do the setup.